libraries/protocol/include/graphene/protocol/types.hpp

```diff
@@ -77,9 +77,18 @@ namespace fc { \
 template<> struct reflector<TYPE> {\
```

Comment:
While you're at it, why not move `FC_REFLECT_DERIVED_NO_TYPENAME` into fc/reflect?

Reply:
Good call, will do

Reply:
https://github.com/bitshares/bitshares-fc/pull/160